bril-rs/brillvm/runtime/build.rs

```diff
@@ -0,0 +1,11 @@
+fn main() {
```

Comment:
What does this function do? We're using this in `eggcc` and ended up copying over `runtime` to our repo, but aren't sure what this is for.